converters/toZigbee.js

```diff
@@ -7409,6 +7409,63 @@ const converters = {
             }
         },
     },
+    inovelli_led_effect: {
```

Comment:
Could you also move these to the `tzLocal` of `inovelli.js`?

Reply:
Done.